tests/monitors/elasticsearch/elasticsearch_test.py

```diff
@@ -144,62 +127,118 @@ def test_elasticsearch_with_threadpool():
             assert not has_log_message(get_output().lower(), "error"), "error found in agent output!"
 
 
+DEFAULT_METRICS = [
```

Comment:
Derive this from the metadata.yaml based on the `included` flag.  See https://github.com/signalfx/signalfx-agent/blob/master/tests/helpers/util.py#L281

Reply:
Done. Added a method in util to return a list of metric names. Did the same for monitor dimensions